### ISSUE TITLE
git clone error: ssh: handshake failed: ssh: unable to authenticate

### ISSUE THREAD

**Issue by yawboateng** [fluxcd/toolkit#251]
am using an existing repo, but getting a handshake failed error even though i see commits with the yaml files and a dir `gitops-system` in the repo.

```
gotk bootstrap github --owner=<org-name> --repository=<private-repo-name> --team=platform --path=bernard

[INFO]  Downloading metadata https://api.github.com/repos/fluxcd/toolkit/releases/latest
[INFO]  Using 0.0.28 as release
[INFO]  Downloading hash https://github.com/fluxcd/toolkit/releases/download/v0.0.28/toolkit_0.0.28_checksums.txt
[INFO]  Downloading binary https://github.com/fluxcd/toolkit/releases/download/v0.0.28/gotk_0.0.28_darwin_amd64.tar.gz
[INFO]  Verifying binary download
[INFO]  Installing gotk to /usr/local/bin/gotk
► connecting to github.com
✔ repository cloned
✚ generating manifests
✔ components manifests pushed
► installing components in gitops-system namespace
namespace/gitops-system unchanged
customresourcedefinition.apiextensions.k8s.io/alerts.notification.toolkit.fluxcd.io configured
customresourcedefinition.apiextensions.k8s.io/gitrepositories.source.toolkit.fluxcd.io configured
customresourcedefinition.apiextensions.k8s.io/helmcharts.source.toolkit.fluxcd.io configured
customresourcedefinition.apiextensions.k8s.io/helmreleases.helm.toolkit.fluxcd.io configured
customresourcedefinition.apiextensions.k8s.io/helmrepositories.source.toolkit.fluxcd.io configured
customresourcedefinition.apiextensions.k8s.io/providers.notification.toolkit.fluxcd.io configured
customresourcedefinition.apiextensions.k8s.io/receivers.notification.toolkit.fluxcd.io configured
role.rbac.authorization.k8s.io/crd-controller-gitops-system unchanged
rolebinding.rbac.authorization.k8s.io/crd-controller-gitops-system unchanged
clusterrolebinding.rbac.authorization.k8s.io/cluster-reconciler-gitops-system unchanged
service/notification-controller unchanged
service/source-controller unchanged
service/webhook-receiver unchanged
deployment.apps/helm-controller configured
deployment.apps/notification-controller configured
deployment.apps/source-controller configured
networkpolicy.networking.k8s.io/deny-ingress unchanged
deployment "source-controller" successfully rolled out
deployment "helm-controller" successfully rolled out
deployment "notification-controller" successfully rolled out
✔ install completed
► generating sync manifests
✔ sync manifests pushed
► applying sync manifests
◎ waiting for cluster sync
✗ git clone error: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain
```

I also tried delete everything however, this failure seems to cause the namespace deletion to hung forever

**Comment by yawboateng** [comment]
so deployment works fine in personal repo. i only get the handshake error in an existing private repo that belongs to an org that am just a member of

**Comment by stefanprodan** [comment]
> I only get the handshake error in an existing private repo that belongs to an org that am just a member of

Are you able to create deploy keys in that repo?

> I also tried delete everything however, this failure seems to cause the namespace deletion to hung forever

This was a bug in `gotk uninstall`, fixed in #254

**Comment by yawboateng** [comment]
> > I only get the handshake error in an existing private repo that belongs to an org that am just a member of
> 
> Are you able to create deploy keys in that repo?

ah, i see the issue, i dont have full permission to that repo. just noticed the deploy keys step also failed

> > I also tried delete everything however, this failure seems to cause the namespace deletion to hung forever
> 
> This was a bug in `gotk uninstall`, fixed in #254

ok, thanks. we can close this now
